app/locale/fi.coffee

```diff
@@ -161,10 +161,10 @@ module.exports = nativeDescription: "suomi", englishDescription: "Finnish", tran
     view_progress: "Tarkastele edistymistäni"
     go_to_courses: "Siirry omiin kursseihin"
     want_coco: "Haluatko CodeCombat:in kouluusi?"
-#    educator: "Educator"
-#    student: "Student"
+    educator: "Opettaja" # {change}
```

Comment:
what is # {change} here? Do you intend to change translation or it's a comment? Same question for below changes

Reply:
I must've misinterpreted the translation instructions where it stated to add # {change} when editing the en.coffee file. I've removed those instances and committed again.